src/renderer/store/actions.js

```diff
@@ -476,13 +476,31 @@ async function fetchTradeHistory({ state, commit }, { marketName, isRequestSilen
 }
 
 async function fetchOrderHistory({ commit }, { isRequestSilent }) {
-  let orders;
+  let orderHistory;
   commit(isRequestSilent ? 'startSilentRequest' : 'startRequest',
     { identifier: 'fetchOrderHistory' });
 
+  const currentNetwork = network.getSelectedNetwork();
+  const currentWallet = wallets.getCurrentWallet();
+  const orderHistoryStorageKey
```

Comment:
I thought about it some more, and for now, I am thinking maybe storing order history in memory is good enough. In the case that we have some scenario that we need to resync the chain to fix order history, I'd like a restart of the wallet to still clean things up for users. Maybe we comment out the code that caches it using pouchdb for now.

Reply:
Actually adding an ability to clear caches in settings might be a better way; and we can continue to cache.